Source/WebCore/rendering/RenderVideo.cpp

```diff
@@ -294,6 +296,15 @@ void RenderVideo::updateFromElement()
     updatePlayer();
 }
 
+bool RenderVideo::inFullscreenElementOrVideoFullscreen() const
```

Comment:
should this be "inElementOrVideoFullscreen"? I don't "in fullscreen element" is right. "has fullscreen element" or "in element fullscreen", yes, but not that, right?